sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/state/StatePartitionReader.scala

```diff
@@ -161,6 +188,16 @@ class StatePartitionReader(
     row.update(2, partition.partition)
     row
   }
+
+  private def unifyStateRowPairWithTTL(pair: (UnsafeRow, UnsafeRow)): InternalRow = {
```

Comment:
Shall we place this function and the row schema definition (code [here](https://github.com/apache/spark/pull/47574/files#diff-d65c7a77b70d4f549969ddd852d83e0a337bfa8c22e1b0a76f8e1ad5c96982d8R187)) into one single class/object so that all following TWS state vars can define their schema/unify function in the class? This way we can probably have better code readability for future List/Map state reader implementations.

Reply:
Done